gateway-protocol/src/main/proto/gateway.proto

```diff
@@ -770,4 +780,25 @@ service Gateway {
   rpc ModifyProcessInstance (ModifyProcessInstanceRequest) returns (ModifyProcessInstanceResponse) {
 
   }
+
+  /*
+    Deletes a resource from the state. Once a resource has been deleted it cannot
+    be recovered. If the resource needs to be available again, a new deployment
+    of the resource is required.
+
+    Deleting a process will cancel any running instances of this process
+    definition. Deleting the latest version of a process will cause the previous
+    version to become the latest version.
+
+    Deleting a decision requirement definitions could cause incidents in process
+    instances referencing these decisions in a business rule task.
```

Comment:
🔧 With #11037 completed, we should clarify similar behavior as we do with processes. So what happens when the decision is deleted, and the user tries to evaluate it directly.